### ISSUE TITLE
[Feature]: Allow Date field to work as time field

### ISSUE THREAD

**Issue by peter-emad99** [radix-vue/radix-vue#1313]
### Describe the feature

i couldn't find  a way to use date field as time input if there is a way can we add examples to the documentation

### Additional information

- [ ] I intend to submit a PR for this feature.
- [ ] I have already implemented and/or tested this feature.

**Comment by zernonia** [comment]
Yeah currently it's abit difficult to construct time field using date field currently. You can technically hide the date segment and populate it automatically to today's date. 

Since this was previously mentioned here (https://github.com/radix-vue/radix-vue/issues/900), I'll close this so we can track the new component in that ticket ya 😁
